client/views/teams/contextualBar/TeamChannels.js

```diff
@@ -149,7 +157,7 @@ const TeamChannels = ({ teamId }) => {
 			items: roomsDated,
 			itemCount: total,
 		};
-	}, [roomListEndpoint, teamId]);
+	}, [roomListEndpoint, teamId, text, type]);
```

Comment:
Currently, we're getting an infinite loop! 
We should isolate the params and wrap them on a `useMemo` and try to use debounce on the text input to do fewer requests when typing

Reply:
Don't know if it's due to this, but during my testing, the filters did not work at all

Reply:
The problem was related to Virtuoso endReached... @gabriellsh you don't see the diference because of no reactivity on the component, that problem was related to another issue...

Reply:
@tiagoevanp Are you sure? For me, the infinite loop seems to be a problem with the lifecycle. About the reactivity.. I don't think so, you can take a look at the `RoomsMembers` it should update the list when typing.

Reply:
@dougfabris How?